### PR TITLE
identify_kinematic_chains-with-branches

### PR DESCRIPTION
**Pull request by Lars-DHondt-KUL**
## Description

Updated the function that detects kinematic chains for the limbs such that it can handle kinematic chains with branches (i.e. one segment can now have more than one distal segment).

## Motivation and Context

Code can now use wider range of models.

## How Has This Been Tested?

Not. qB)

## Suggested tests for reviewers

./Tests/test_PredSim.m

### CODE REVIEW

File: PreProcessing/identify_kinematic_chains.m [hunk 1]
```diff
@@ -346,17 +346,28 @@
     joints_chain = [];
```
Comment (dgupta7) [inline PreProcessing/identify_kinematic_chains.m:346]: Can you add a function header here?

**Review by dgupta7 — COMMENTED** [review]
Can you give a small example of the structure of the variables involved? I am wondering how the joint_table structure looks like and how the resultant joints_chain and coords_chain looks like.

File: PreProcessing/identify_kinematic_chains.m (hunk 1 above)
Comment (Lars-DHondt-KUL) [reply PreProcessing/identify_kinematic_chains.m:377]: Added header that includes the structure of the variables.

Comment (dgupta7) [reply PreProcessing/identify_kinematic_chains.m:377]: Thanks!

**Review by dgupta7 — APPROVED** [review]
Thanks for the updates!

**Pull request merged**